mongospecs/attrs.py

```
@@ -7,9 +7,10 @@
 from bson import ObjectId
 from pymongo import MongoClient
 
-from .base import SpecBase, SpecDocumentType, SubSpecBase
-from .empty import Empty
-from .se import MongoEncoder, mongo_dec_hook
+from .base import SpecBase, SubSpecBase
+from .types import SpecDocumentType
```

Ruff (0.8.2)

`.types.SpecDocumentType` imported but unused

Remove unused import: `.types.SpecDocumentType`

(F401)